product_docs/docs/pgd/5/consistency/column-level-conflicts/05_clc_crdt.mdx

```diff
@@ -0,0 +1,8 @@
+---
+navTitle: Using CRDT data types
+title: Handling column-level conflicts using CRDT data types
+redirects:
+  - /pgd/latest/bdr/column-level-conflicts/
+---
+
+By default, column-level conflict resolution picks the value with a higher timestamp and discards the other one. You can, however, reconcile the conflict in different, more elaborate ways. For example, you can use CRDT types that allow merging the conflicting values without discarding any information.
```

Comment:
Overly short, doesn't link to CRDT datatypes. Probably wants to be integrated in another page where resolutions are discussed?

Reply:
added the link (although of course it will change again when I do the splitting up of CRDT data types 🤷)

Reply:
I tried integrating at the end of the overview on column-level conflict resolution.